taxi/src/main/java/com/epam/training/taxi/account/UnregisteredAccount.java

```diff
@@ -0,0 +1,19 @@
+package com.epam.training.taxi.account;
+
+public final class UnregisteredAccount implements Account {
+
+    @Override
+    public Long getAccountId() {
+        return null;
```

Comment:
Be careful when returning `null`s due to NPEs. If you must, I'd suggest using `Optional` as a more safe option.